Snippets/isns.sqlpromptsnippet

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets>
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>isns</Title>
+      <Shortcut>isns</Shortcut>
+      <Description>ISNULL STRING: WRAP CURRENT SELECTION IN ISNULL OPERATOR for a string</Description>

```

Comment:
Surely this works for things other than STRINGs too? I'd regularly use this for NUMERIC fields where I want to convert NULL values to zero, or some other value. 

Please remove the "for a string" part from here, and from the 'list' file.